### PR TITLE
Big sprite cleanup

### PR DESCRIPTION
**Pull request by thgvr**
## About The Pull Request
Removing a bunch of unused stuff and cleaning up/moving/organizing files.

## Changelog

:cl:
add: Added new things
add: Added more things
del: Removed old things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

**Comment by goober3** [comment]
i'm disappointed at the lack of big sprites in this PR

File: code/modules/mob/living/simple_animal/hostile/human/skeleton.dm [hunk 1]
```diff
@@ -48,7 +48,6 @@
 	deathmessage = "collapses into a pile of bones, its gear clanging as it hits the ground!"
 	loot = list(/obj/effect/decal/remains/human,
 				/obj/item/clothing/suit/armor/witchhunter,
-				/obj/item/clothing/head/witchunter,
```
Comment (FalloutFalcon) [inline code/modules/mob/living/simple_animal/hostile/human/skeleton.dm:51]: Why are we leaving in the witchhunter amour but not that hat. The srm uses the amour.

File: _maps/shuttles/independent/independent_beluga.dmm [hunk 2]
```diff
@@ -666,7 +666,7 @@
 /obj/item/clothing/glasses/sunglasses/big,
 /obj/item/clothing/suit/toggle/lawyer/burgundy,
 /obj/item/clothing/suit/toggle/lawyer/navy,
-/obj/item/clothing/head/hopcap,
+/obj/item/clothing/head/nanotrasen/officer,
```
Comment (Sadhorizon) [inline _maps/shuttles/independent/independent_beluga.dmm:669]: They... aren't NT. Just kill it completely probably.

File: code/modules/mob/living/simple_animal/hostile/human/skeleton.dm (hunk 1 above)
Comment (thgvr) [reply code/modules/mob/living/simple_animal/hostile/human/skeleton.dm:51]: i really hope they don't

Comment (thgvr) [reply code/modules/mob/living/simple_animal/hostile/human/skeleton.dm:51]: also im not done

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 3]
```diff
@@ -287,57 +191,6 @@
...
-	AddComponent(/datum/component/squeak, list('sound/machines/clockcult/integration_cog_install.ogg' = 1, 'sound/magic/clockwork/fellowship_armory.ogg' = 1), 50, extrarange = SHORT_RANGE_SOUND_EXTRARANGE)
-
-/obj/item/clothing/shoes/wheelys
-	name = "Wheely-Heels"
-	desc = "Uses patented retractable wheel technology. Never sacrifice speed for style - not that this provides much of either." //Thanks Fel
-	icon_state = "wheelys"
-	item_state = "wheelys"
-	actions_types = list(/datum/action/item_action/wheelys)
-	var/wheelToggle = FALSE //False means wheels are not popped out
-	var/obj/vehicle/ridden/scooter/wheelys/W
```
Comment (FalloutFalcon) [inline code/modules/clothing/shoes/miscellaneous.dm:309]: I humbly plead that the are not removed.

**Comment by thgvr** [comment]
this wasn't ready to merge. i mean look at the changelog

**Comment by Apogee-dev** [comment]
ough. do we need to revert or

**Comment by generalthrax** [comment]
Probably shouldve drafted this

**Comment by thgvr** [comment]
probably (I didn't test it)

**Comment by Sun-Soaked** [comment]
do you want me to hit the button

**Comment by thgvr** [comment]
not really

**Pull request merged**